srcpkgs/python3-trimesh/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'python3-trimesh'
 pkgname=python3-trimesh
-version=3.8.11
+version=3.8.15
 revision=1
 wrksrc="trimesh-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-numpy"
 depends="python3-numpy python3-scipy python3-networkx python3-lxml python3-pyglet python3-shapely python3-rtree python3-Pillow python3-svg.path python3-jsonschema python3-psutil python3-chardet python3-sympy python3-msgpack python3-pycollada python3-requests"
```

Comment:
Would you mind splitting this line at ~80 columns?

Reply:
fixed